src/main/kotlin/calculator/Param.kt

```diff
@@ -0,0 +1,42 @@
+package calculator
+
+class Param(text: String) {
+    val value: Int
+
+    constructor(intValue: Int) : this(intValue.toString())
+
+    init {
+        if (text.matches(PARAM_NOT_MATCH_VALIDATOR)) {
+            throw RuntimeException("숫자가 아닌 입력은 들어올 수 없습니다. (음수도 안됩니다!)")
+        }
```

Comment:
```suggestion
        val i = if (text.isBlank()) {
            0
        } else {
            text.toIntOrNull() ?: throw RuntimeException("숫자가 아닌 입력은 들어올 수 없습니다.")
        }
        require(i >= 0) { "음수는 불가능합니다" }
        
        value = i
```
Regex를 사용하는 방법도 있지만, 검증된 코틀린 내장 API를 사용하는 것도 좋은 방법인 것 같아요!